### PR TITLE
[aws-for-fluent-bit] Add log_retention_days to CloudWatch plugin

### PR DESCRIPTION
**Pull request by mickeypash**
The `aws-for-fluent-bit` chart was not exposing the configuration for`log_retention_days` provided by the original [CloudWatch Plugin](https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit). 

This change attempts to address this, thus allowing users of the chart to set the number of days they would like their log streams to be kept for.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: stable/aws-for-fluent-bit/README.md
Comment (mforutan) [inline stable/aws-for-fluent-bit/README.md:52]: Can you please update the [values.yaml file](https://github.com/aws/eks-charts/blob/12d2ba5ace78a5cf62bbcf2e5ab97c58f62f0c69/stable/aws-for-fluent-bit/values.yaml#L53-L65) with this new variable?

**Comment by mickeypash** [comment]
Any other feedback?

**Review by mforutan — APPROVED** [review]
LGTM but we need to wait for an admin to merge it in

**Comment by mickeypash** [comment]
Is there anything I can do to help get this prioritised or merged?

**Review by fawadkhaliq — APPROVED** [review]

**Comment by mickeypash** [comment]
So I see a new version has been released for the `eks-charts` package. 
But I can't see these changes reflected here: https://artifacthub.io/packages/helm/aws/aws-for-fluent-bit

**Comment by mforutan** [comment]
@mickeypash that new release just happened before this merge so we have to wait for the next release to see these changes

**Pull request merged**
